tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_search_bug.py

```diff
@@ -0,0 +1,28 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.bzlite.app import BugzillaLite
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+
+class TestSearchBug(GaiaTestCase):
+	def setUp(self):
+		GaiaTestCase.setUp(self)
+		self.connect_to_local_area_network()
```

Comment:
@JohanLorenzo I need to add uninstall function here before the installation of the staging App